### DISCUSSION TITLE
Problema ao autorizar GitHub no App Service do Azure

### DISCUSSION THREAD

**Discussion by Gilthayllor** [balta-io/forum#258] [Tira Dúvidas] [answered]
Olá, estou no curso **Fundamentos do Azure, Git, GitHub e DevOps**, na aula **Publicando a API**. Para prosseguir com a integração entre o GitHub e Azure, é necessário autorizar o GitHub no app service, mas sempre que clico no botão **Authorize** uma nova janela do navegador se abre com o erro 404. Já fiz o teste em 3 navegadores diferentes e o erro persiste. Não encontrei nada na internet sobre como resolver e o que pode estar acontecendo de errado.

![image](https://user-images.githubusercontent.com/38956639/217686284-f0f491f9-8e05-460a-b2ed-f5541a15f331.png)

![image](https://user-images.githubusercontent.com/38956639/217685853-f53a5291-45d5-41a5-966c-1f0be9aee17e.png)

**Comment by andrebaltieri** [comment]
Bom dia, @Gilthayllor, como vai?

Tentou fazer em uma aba anônima?

**Comment by Gilthayllor** [comment]
Bom dia, Balta!

Tentei sim, após o login no GitHub sou redirecionado para a página do erro, e, no console recebo o erro 401 junto da mensagem "Failed to load resource: the functions.azure.com/api/github/get user".

Pelo que vi, o problema já aconteceu antes com outros usuários como nesses relatos abaixo:

https://github.com/MicrosoftDocs/azure-docs/issues/28750

https://learn.microsoft.com/en-us/answers/questions/66288/azure-app-service-authorized-github-not-working



________________________________

**Reply by andrebaltieri** [reply]
Bom dia, @Gilthayllor, como vai?
Estranho, eu nunca tive este problema. Você tem habilitado autenticação dois fatores? Seria uma boa habilitar e testar.

Em adicional, talvez tentar abrir um chamado direto no GitHub para ver se tem algo errado com sua conta.

**Comment by Gilthayllor** [comment]
Bom dia, Balta!

Já possuo a autenticação de dois fatores habilitada.

Vou entrar em contato com o suporte do GitHub mesmo, já tentei pelo suporte do Azure, mas não conseguiram resolver.
Atualizo aqui caso resolvam, obrigado!

________________________________

**Reply by andrebaltieri** [reply]
Beleza @Gilthayllor, se puder nos manter informados aqui... agradecemos 🚀

**Comment by Gilthayllor** [comment, marked as answer]
Boa tarde!

Enviei um e-mail para **azcommunity@microsoft.com** relatando o problema que estava acontecendo, juntamente com o **Subscription ID** do Azure. Apesar de não receber nenhum e-mail como resposta, consegui autorizar normalmente hoje.

Espero que esta solução ajude alguém no futuro caso isso aconteça novamente.

**Reply by andrebaltieri** [reply]
😵
Que bom que deu certo e muito obrigado por compartilhar aqui.
